### PR TITLE
Remove peddy dependency

### PR DESCRIPTION
**Pull request by vivbak**
As suggested https://github.com/populationgenomics/images/pull/8

### CODE REVIEW

File: scripts/create_test_subset.py [hunk 1]
```diff
@@ -104,19 +103,17 @@ def main(
...
-        families = random.sample(families, families_n)
-        logger.info(f'After subsetting to {len(families)} families:')
-        _print_fam_stats(families)
-        sample_ids = []
-        for fam in families:
-            for s in fam.samples:
-                sample_ids.append(sample_id_by_participant_id[s.sample_id])
+        fams = fapi.get_families(project)
+        all_families = [family['id'] for family in fams]
+        families = random.sample(all_families, families_n)
```
Comment (illusional) [inline scripts/create_test_subset.py:109]: Missing a `_print_fam_stats` call here

File: scripts/create_test_subset.py [hunk 2]
```diff
@@ -104,19 +103,17 @@ def main(
...
+        pedigree = fapi.get_pedigree(project=project, internal_family_ids=families)
+        p_ids = [ped['individual_id'] for ped in pedigree]
+        sample_ids = [
+            sample
+            for (participant, sample) in sample_id_by_participant_id.items()
+            if participant in p_ids
+        ]
         samples = [s for s in all_samples if s['id'] in sample_ids]
```
Comment (illusional) [inline scripts/create_test_subset.py:117]: calling `in` on a non-hashed type is an `O(n) operation, and in a loop like this is `O(n**2)`, you might consider making a set version of sample_ids to do a contains check.

**Review by illusional — COMMENTED** [review]
I started reviewing this before I realised it was a draft, so early feedback.

File: scripts/create_test_subset.py (hunk 1 above)
Comment (vivbak) [reply scripts/create_test_subset.py:109]: Ah yes, would have to reimplement that function to work with the new input format. Is the print useful to have?

Comment (vivbak) [reply scripts/create_test_subset.py:109]: Fixed in the latest :)

File: scripts/create_test_subset.py (hunk 2 above)
Comment (vivbak) [reply scripts/create_test_subset.py]: I think I've followed your suggestion in the latest push but let me know if I misinterpreted

File: scripts/create_test_subset.py [hunk 3]
```diff
@@ -298,9 +297,10 @@ def _validate_opts(samples_n, families_n) -> Tuple[Optional[int], Optional[int]]
 
 
 def _print_fam_stats(families: List):
+    family_sizes = Counter([fam['family_id'] for fam in families])
     fam_by_size: typing.Counter[int] = Counter()
-    for fam in families:
-        fam_by_size[len(fam.samples)] += 1
+    for fam in family_sizes:
```
Comment (illusional) [inline scripts/create_test_subset.py:302]:
```suggestion
    # determine number of singles, duos, trios, etc
    for fam in family_sizes:
```

**Review by illusional — APPROVED** [review]

**Pull request merged**